content/work/moztrap.rst

```diff
@@ -18,10 +18,10 @@ tasks:
   - Project Architecture
   - Responsive Styles
 quotes:
-  - text: 'I consider MozTrap to be one of the best
+  - text: 'I consider <a href="/work/">Mozilla’s MozTrap</a> to be one of the best
```

Comment:
This is a bit weird to when seeing it on the work page, the link just refreshes the page.  I realize the quotes are displayed everywhere, but maybe the link isn't necessary in the quote itself?

And right now we aren't linking to this work/moztrap page from anywhere yet, are we?